### PR TITLE
MongoDbManager for RBAC

### PR DESCRIPTION
**Pull request by dkhlystov**

### CODE REVIEW

File: rbac/MongoDbManager.php [hunk 1]
```diff
@@ -0,0 +1,967 @@
...
+     * @var Connection|array|string the DB connection object or the application component ID of the DB connection.
+     * After the MongoDbManager object is created, if you want to change this property, you should only assign it
+     * with a DB connection object.
+     * Starting from version 2.0.2, this can also be a configuration array for creating the object.
+     */
+    public $db = 'mongodb';
+    /**
+     * @var string the name of the table storing authorization items. Defaults to "auth_item".
+     */
+    public $itemTable = 'auth_item';
```
Comment (klimov-paul) [inline rbac/MongoDbManager.php]: This is inappropriate name. It should be `itemCollection`.

File: rbac/MongoDbManager.php [hunk 2]
```diff
@@ -0,0 +1,967 @@
...
+    /**
+     * @var string the name of the table storing authorization item hierarchy. Defaults to "auth_item_child".
+     */
+    public $itemChildTable = 'auth_item_child';
```
Comment (klimov-paul) [inline rbac/MongoDbManager.php]: Same here and below.

File: rbac/MongoDbManager.php [hunk 3]
```diff
@@ -0,0 +1,967 @@
+<?php
+
```
Comment (samdark) [inline rbac/MongoDbManager.php]: Standard class header missing.

**Comment by klimov-paul** [comment]
Resolved via #110

**Pull request closed without merging**